packages/manager/apps/hub/src/index.html

```diff
@@ -2,6 +2,7 @@
 <html>
     <head>
         <title>ManagerHub App</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
Is it needed for the scope of this PR ?

Reply:
Its not required. Used this while testing the component out. I had pushed it since it would be required anyways. Removed it now :)